libs/blocks/brick/brick.js

```diff
@@ -1,11 +1,13 @@
-import { decorateTextOverrides, decorateBlockText, decorateBlockBg, decorateIconStack, decorateButtons } from '../../utils/decorate.js';
+import {
+  // eslint-disable-next-line max-len
```

Comment:
I'd say it's better to simply add returns after each method than add 35 characters to disable a rule. Would make things more readable, would not go against any formatting rules and would not impact the size 🤷

Reply:
Agree, simple return is better. I've updated this in bricks.js and hero-marquee.js